Jobsway2goMvc/Data/ApplicationDbContext.cs

```diff
@@ -17,6 +18,7 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : ba
         public DbSet<Collection> Collections { get; set; }
         public DbSet<Job> Jobs { get; set; }
         public DbSet<JobCategory> JobCategories { get; set; }
+        public DbSet<Jobsway2goMvc.Models.ViewModel.UserRolesViewModel> Role { get; set; }
         //public DbSet<Event> Events { get; set; }
         //public DbSet<Notification> Notifications { get; set; }
```

Comment:
Use have reference with `using`, remove whole layer path in DbSet. DbSet<UserRolesViewModel> is enough.